docs/proposed/psa-driver-interface.md

```diff
@@ -675,6 +696,14 @@ ECC key pairs are represented as the private key value only. The public key need
 
 The specification doesn't mention when the public key might be calculated. The core may calculate it on creation, on demand, or anything in between. Opaque drivers have a choice of storing the public key in the key context or calculating it on demand and can convey whether the core should store the public key with the `"store_public_key"` property. Is this good enough or should the specification include non-functional requirements?
 
+#### Symmetric key validation with transparent drivers
+
+Should the entry point be called for symmetric keys as well?
```

Comment:
If we expect that the check in drivers would always be the same I would say no, better to do it once for all in PSA core.

Reply:
+1